SwissTransferFeatures/TransferList/TransferCell.swift

```diff
@@ -49,7 +49,12 @@ struct TransferCell: View {
                 SeparatedItemsView {
                     Text(transfer.sizeUploaded, format: .defaultByteCount)
                 } rhs: {
-                    Text(transfer.expirationDateTimestamp.formatted(.expiring))
```

Comment:
We might be able to simplify the `ExpiringDateFormat` FormatStyle because it was the only use case of `expiring`.